modules/consentManagement.js

```diff
@@ -147,92 +102,39 @@ function lookupIabConsent({onSuccess, onError, width, height}) {
 
   if (isFn(cmpFunction)) {
     logInfo('Detected CMP API is directly accessible, calling it now...');
-    if (cmpVersion === 1) {
-      cmpFunction('getConsentData', null, v1CallbackHandler.consentDataCallback);
-      cmpFunction('getVendorConsents', null, v1CallbackHandler.vendorConsentsCallback);
-    } else if (cmpVersion === 2) {
-      cmpFunction('addEventListener', cmpVersion, v2CmpResponseCallback);
-    }
-  } else if (cmpVersion === 1 && inASafeFrame() && typeof window.$sf.ext.cmp === 'function') {
-    // this safeframe workflow is only supported with TCF v1 spec; the v2 recommends to use the iframe postMessage route instead (even if you are in a safeframe).
-    logInfo('Detected Prebid.js is encased in a SafeFrame and CMP is registered, calling it now...');
-    callCmpWhileInSafeFrame('getConsentData', v1CallbackHandler.consentDataCallback);
-    callCmpWhileInSafeFrame('getVendorConsents', v1CallbackHandler.vendorConsentsCallback);
+    cmpFunction('addEventListener', CMP_VERSION, v2CmpResponseCallback);
```

Comment:
The function named `v2CmpResponseCallback` should be renamed as `cmpResponseCallback` (as you did for  ~`checkV2Data`~  ->  `checkData`)